test/hotspot/jtreg/compiler/vectorization/runner/ArrayTypeConvertTest.java

```diff
@@ -24,6 +24,7 @@
 
 /*
  * @test
+ * @bug 8183390 8340010 8342095
```

Comment:
Review note: This test was missing a `@bug` annotation, so I went through the history and added the ones that substantially modified the test.